x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/ilm/AllocateActionTests.java

```diff
@@ -114,7 +114,7 @@ public void testInvalidNumberOfReplicas() {
         Map<String, String> map = new HashMap<>();
         int numIncludes = randomIntBetween(minEntries, maxEntries);
         for (int i = 0; i < numIncludes; i++) {
-            map.put(randomAlphaOfLengthBetween(1, 20), randomAlphaOfLengthBetween(1, 20));
+            map.put(randomAlphaOfLengthBetween(2, 20), randomAlphaOfLengthBetween(2, 20));
```

Comment:
what's the reason behind this change?

Reply:
So the test that actually caused the failure generated `n` as the key in the map, by increasing it to at least 2, we decrease the chance of collisions. The other changes make sure we don't have collisions, but this at least decreases the amount of retries since it's less likely to collide.